chapters/chap02-sec01.tex

```diff
@@ -5,8 +5,10 @@ \section{\zeth~Data Types}\label{zeth-protocol:zeth-data-types}
 We begin by describing, and giving intuition about, the data types (see~\cref{preliminaries:structured-data}) used in \zeth.
 We follow some design rationale from \zerocash~\cite{sasson2014zerocash}, and \zcash~\cite{zcashprotocol} in order to prevent the transaction malleability attack, and the Faerie Gold attack\cite[Section 8.4]{zcashprotocol}. We refer the reader to~\cref{appendix:trnm} for more details.
 
+In what follows \Curve~represents a curve with scalar field $\FFx{\rCURVE}$, satisfying the requirements of \cref{instantiation:zksnark}. The specification is described in terms of this generic curve, with examples and notes relating to specific instances of interest (namely \BNCurve~and \BLSCurve, see \cref{chap:instantiation}).
```

Comment:
As well as this introduction, chap03-base (instantiation sectionn introduction) has been expanded to explain that concrete values / instances are given, where it makes sense, for the two curves of interest.

@AntoineRondelet  This seemed like an easy way to handle this, but I can see several other possibilities (such as a whole chap03-sec?? instantiation section for curves, etc).  Let me know what you think.